examples/scripts/rl_enjoy.py

```diff
@@ -25,6 +25,43 @@
 import torch
 
 from evotorch.neuroevolution.net.rl import reset_env, take_step_in_env
+from evotorch.tools.versionchecking import check_version
+
+new_render_api = True
+
+gym_ver = check_version(gym, 2)
+
+if gym_ver is not None:
+    a, b = gym_ver
+    if (a == 0) and (b < 26):
+        new_render_api = False
+
+
+def make_env_for_rendering(*args, **kwargs):
```

Comment:
Since these are example scripts. It would be helpful to have some more commentary about what these functions regarding rendering and recording are doing / why they are needed.

Reply:
Agreed, thanks. I added docstrings which clarifies the actions taken by the function when the gym version is newer than or equal to 0.26.